src/com_tjvendors/admin/models/fields/paymentgateway.php

```diff
@@ -55,11 +55,14 @@ protected function getOptions()
 		if (!empty($client))
 		{
 			$com_params = JComponentHelper::getParams($client);
-			$gateways = $com_params['gateways'];
+			$gateways = (array) $com_params->get('gateways');
 
-			foreach ($gateways as $detail)
+			if (!empty($gateways))
```

Comment:
If you have typecasted this as an array, no need for an if condition for foreach.